opencv/c_extension/fast_image_processing.cpp

```diff
@@ -0,0 +1,229 @@
+#include <string>
+#include <iostream>
+#include <vector>
+#include <math.h> // atan
+#include <cmath> // std::abs
+
+#include <pybind11/pybind11.h>
+
+#include <Eigen/Core>
+#include <Eigen/Dense>
+
+#include <opencv2/core/core.hpp>
+#include <opencv2/core/eigen.hpp>
+#include <opencv2/highgui/highgui.hpp>
+#include <opencv2/imgproc.hpp>
+
+#define CONVERT_TO_INT true
+#define MAX_WIDTH 320
+#define PI 3.14159265
```

Comment:
In OpenCV there is already a define
#define CV_PI   3.1415926535897932384626433832795

Reply:
ok, i'll replace that